quickstart_guides/ruby.md

```diff
@@ -2,3 +2,84 @@ address: support.aptible.com/quickstart/ruby
 
 ### Ruby Quickstart
 
+This guide assumes you have:   
+- An Aptible account,  
+- An SSH key associated with your Aptible user account, and  
+- The Aptible command line tool installed
+
+#### 1. Provision Your App  
+Tell the Aptible API that you want to provision an application. Until you push code and trigger a build, Aptible uses this as a placeholder.
+
+##### Command line
+Use the `apps:create` command: `aptible apps:create HANDLE`
+
+For example: 
+```
+aptible apps:create my-example-app
+```
+
+If you have multiple accounts, you will need to specify `--account <account_handle>`. Each account handle is listed next to the account name in the Aptible Dashboard.
+
+For example: 
+```
+aptible apps:create example-app --account myco-development
+```
+
+##### Aptible Dashboard  
+From the [Aptible dashboard](https://dashboard.aptible.com/apps), pick an account to provision the app into, select "Create App," and choose a handle.
+
+#### 2. Add a Dockerfile
```

Comment:
We can pretty easily eliminate this, by inferring a default buildstep Dockerfile if none is present. What does everyone think: do we want to? cc: @aaw